1/Dockerfile

```diff
@@ -14,12 +14,14 @@ MAINTAINER Ben Parees <bparees@redhat.com>
 
 ENV JENKINS_VERSION=1.609 \
     HOME=/var/lib/jenkins \
-    JENKINS_HOME=/var/lib/jenkins
+    JENKINS_HOME=/var/lib/jenkins \
+    STI_SCRIPTS_URL=image:///usr/libexec/s2i
```

Comment:
i think we can safely remove this at this point, what do you think?  we deprecated it in s2i quite a while ago at this point.